mlc_config.json

```diff
@@ -0,0 +1,6 @@
+// 429 Too Many Requests
+// This is the typical github answer after too much requests, so I would like to ignore this status when it happens.
```

Comment:
Great to have comments 👍  but I think this will cause the file to be invalid

Reply:
@eddiejaoude 
My bad, you're definitely right 👍🏻